nuntium/templates/thread/to.html

```diff
@@ -4,9 +4,9 @@
 {% block content_inner %}
 
     <div class="person-profile">
-        <img class="person-profile__photo" src="http://placehold.it/128x128" width="128" height="128">
+        <img class="person-profile__photo" src="{% if person.image %}{{person.image}}{% else %}http://placehold.it/128x128{% endif %}" width="128" height="128">
```

Comment:
Showing the real image if it is exists is definitely an improvement(!), but I don't think that's a particularly good placeholder to show if there isn't one...